### PR TITLE
Ads/admin ui responsive layout

### PR DESCRIPTION
**Pull request by darrikonn**
layout

### NOTE
~Ignore Checkbox changes since these are changes from https://github.com/island-is/island.is/pull/887~

~Due to stashing of PRs and since this PR is dependent of #887 but unrelated, I don't want to set the base to #887~

### CODE REVIEW

File: apps/air-discount-scheme/web/screens/Admin/Admin.tsx [hunk 1]
```diff
@@ -39,50 +88,94 @@ const Admin: Screen = ({}) => {
...
-            marginBottom={3}
-            borderRadius="standard"
-          >
-            <Box marginBottom={2}>
-              <Typography variant="h4">Aðgerðir</Typography>
+    <GridContainer>
+      <GridRow>
+        <GridColumn
+          span={['12/12', '12/12', '7/12', '8/12', '9/12']}
+          order={[0, 0, 2, 2, 2]}
```
Comment (barabrian) [inline apps/air-discount-scheme/web/screens/Admin/Admin.tsx:95]: I think this should be enough 
```suggestion
          order={[0, 0, 2]}
```

File: apps/air-discount-scheme/web/screens/Admin/Admin.tsx [hunk 2]
```diff
@@ -39,50 +88,94 @@ const Admin: Screen = ({}) => {
...
+        >
+          <Hidden above="sm">
+            {!loading && <Summary flightLegs={flightLegs} />}
+          </Hidden>
+        </GridColumn>
+        <GridColumn
+          span={['12/12', '12/12', '5/12', '4/12', '3/12']}
+          order={[1, 1, 1, 1, 1]}
```
Comment (barabrian) [inline apps/air-discount-scheme/web/screens/Admin/Admin.tsx:103]: I think this should be enough 
```suggestion
          order={1}
```

File: apps/air-discount-scheme/web/screens/Admin/Admin.tsx [hunk 3]
```diff
@@ -39,50 +88,94 @@ const Admin: Screen = ({}) => {
...
-          </Box>
-        </Stack>
-      }
-    />
+          </Stack>
+        </GridColumn>
+
+        <GridColumn
+          span={['12/12', '12/12', '7/12', '8/12', '9/12']}
+          order={[2, 2, 0, 0, 0]}
```
Comment (barabrian) [inline apps/air-discount-scheme/web/screens/Admin/Admin.tsx:137]: I think this should be enough 
```suggestion
          order={[2, 2, 0]}
```

**Review by barabrian — APPROVED** [review]
LGTM, minor comments

**Review by barabrian — APPROVED** [review]

**Review by barabrian — APPROVED** [review]

**Review by dabbeg — APPROVED** [review]
Like a glove!

**Pull request merged**